tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -25,11 +25,13 @@ all: cloud-hypervisor-tarball \
 	firecracker-tarball \
 	kernel-tarball \
 	qemu-tarball \
-	rootfs-image-tarball \
-	rootfs-initrd-tarball \
+	rootfs-tarballs \
 	shim-v2-tarball \
 	virtiofsd-tarball
 
+rootfs-tarballs: rootfs-image-tarball \
+	rootfs-initrd-tarball \
```

Comment:
Hmm. I would think this would still be lead to those two running in parallel, no?

Also, extra `\` by the end pf the line.